### DISCUSSION TITLE
how use the JS `undefined` with Brython

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2496] [Q&A] [answered]
Hi,

I would like to convert a JS script into Brython (3.11.2)

This JS script is using a var type `undefined`

How can I use int in my brython code ?

I've tried 

```python
JSDate = window.Date.new()

#1
JSDate.toLocaleTimeString(undefined)

#2
JSDate.toLocaleTimeString(window.undefined)

#3
JSDate.toLocaleTimeString(None)
```
but nothing works.

I've review https://brython.info/static_doc/3.11/en/jsobjects.html & https://brython.info/static_doc/3.11/en/javascript.html

But I don't figure it out...

Any ideas ?

**Comment by denis-migdal** [comment]
> Hi,
> 
> I would like to convert a JS script into Brython (3.11.2)
> 
> This JS script is using a var type `undefined`
> 
> How can I use int in my brython code ?
> 
> I've tried
> 
> ```python
> JSDate = window.Date.new()
> 
> #1
> JSDate.toLocaleTimeString(undefined)
> 
> #2
> JSDate.toLocaleTimeString(window.undefined)
> 
> #3
> JSDate.toLocaleTimeString(None)
> ```
> 
> but nothing works.
> 
> I've review https://brython.info/static_doc/3.11/en/jsobjects.html & https://brython.info/static_doc/3.11/en/javascript.html
> 
> But I don't figure it out...
> 
> Any ideas ?

Just use `JSDate.toLocaleTimeString()`.
Else you can get undefined from `JavaScript.UNDEFINED` IIRC.

**Reply by BobSquarePants** [reply, marked as answer]
Thank @denis-migdal  ,

with your clues, I've managed to make it works :)

```python
import javascript

print( window.Date.new().toLocaleTimeString(javascript.UNDEFINED,{'hour12':False,'hour':'2-digit','minute':'2-digit'}) )
#output: 22:24 for example :)
```
